frontend/packages/component-library/src/cms/tabGroup/TabGroup.tsx

```diff
@@ -0,0 +1,158 @@
+import classNames from 'classnames';
+import {useMemo} from 'react';
+
+import Accordion from '@/accordion';
+import {AccordionItem} from '@/accordion/Accordion';
+import {LinkButton, LinkButtonProps} from '@/button';
+import Image, {ImageProps} from '@/cms/image';
+import Tabs, {TabModel, TabsProps} from '@/tabs';
+import {BodyThreeText, Heading3, Heading4} from '@/typography';
+
+import moduleStyles from './tabGroup.module.scss';
+
+export interface TabGroupTabModel extends Omit<TabModel, 'tabContent'> {
+  /** Content of the Tab */
+  tabContent: {
+    /** Tab Image props */
+    image: ImageProps;
```

Comment:
Can we exclude the `decoration: shadow` type for the `Image` component here? That should only be used when an image is in a standalone section. (`border` is fine to keep though)

Reply:
I tried updating this, but it then causes regressions by needing to then update test file which feels like not worth the effort and complexity, as we can simply not use `decoration` prop in Contentful at all. That's why I'm going to keep this simple, as is right now for now.

If we agree it's critical - we can update this in a follow up, of course.